.eslintrc.json

```diff
@@ -738,6 +738,7 @@
 					"rxjs/**",
 					"ng2-charts",
 					"chart.js",
+					"plotly.js",
```

Comment:
We shouldn't have both plotly.js and plotly.js-dist-min. Do we really need both? Can we use the existing one?

Reply:
plotly.js only refers to the _@types/plotly.js_ module, and doesn't reference the library which is only installed in the _plotly.js-dist-min_ package. 

As far as I understand, the only difference from using the actual _plotly.js_ library vs the _plotly.js-dist-min_ package is that types would automatically be resolved if we use _plotly.js_. I don't think there are performance benefits or anything else. I can make the switch, but I didn't know if there was a good reason to use  _plotly.js-dist-min_.

Reply:
Yeah this is kind of a weird one. I'm not even sure why we wanted the minified version here - we minify the code ourselves which should include dependent packages. 

But it doesn't look like the plotly.js or the plotly.js-dist packages even have the typings anyways (which makes sense - they're pure javascript libraries). So we'd still need the typings file regardless.

I did check quick and it looks like the plotly.js-dist-min rule here is unnecessary though, it seems the linter doesn't check imports like we're using for plotly.js-dist-min. So you can probably just remove that one.